examples/simple/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "name" {
+  type        = string
+  description = "Name of the router"
+}
+
+variable "network" {
+  type        = string
+  description = "A reference to the network to which this router belongs"
+}
+
+variable "project" {
+  type        = string
+  description = "The GCP project ID"
```

Comment:
The string-version of the integer-version (which I think is also persisted as string) of the ID?

Reply:
So it is the ID field and not the Name field. 
According to google it has some limitations, namely 
> Project ID can have lowercase letters, digits, or hyphens. It must start with a lowercase letter and end with a letter or number.
>The project ID is the globally unique identifier for your project. You cannot change the project ID after the project is created.